falco/templates/daemonset.yaml

```
@@ -196,15 +196,22 @@ spec:
                   name: {{ template "falco.fullname" . }}
                   key: gcp-credentials-data
       {{- end }}
-      {{- if (or .Values.integrations.natsOutput.enabled .Values.integrations.snsOutput.enabled .Values.integrations.pubsubOutput.enabled) }}
+      {{- if (or .Values.integrations.natsOutput.enabled .Values.integrations.snsOutput.enabled .Values.integrations.pubsubOutput.enabled .Values.driverbuilder.initcontainer) }}
```

Why not add just a general way to add an init-container (instead of putting it under the driverbuilder section)?
```suggestion
      {{- if (or .Values.integrations.natsOutput.enabled .Values.integrations.snsOutput.enabled .Values.integrations.pubsubOutput.enabled .Values.initcontainer) }}
```